Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -761,6 +783,28 @@ final class AppDelegateViewModelTests: TestCase {
     }
   }
 
+  func testPresentViewController_UpdateCommentThread_Reply_Success() {
+    self.vm.inputs.applicationDidFinishLaunching(
+      application: UIApplication.shared,
+      launchOptions: [:]
+    )
+
+    withEnvironment(apiService: MockService(fetchCommentRepliesEnvelopeResult: .success(CommentRepliesEnvelope
+        .successfulRepliesTemplate), fetchProjectResponse: .template)) {
+      let url =
+        "https://\(AppEnvironment.current.apiService.serverConfig.webBaseUrl.host ?? "")/projects/fjorden/fjorden-iphone-photography-reinvented/posts/3254626/comments?comment=Q29tbWVudC0zMzY0OTg0MQ%3D%3D&reply=deadbeef"
+
+      let result = self.vm.inputs.applicationOpenUrl(
+        application: UIApplication.shared,
+        url: URL(string: url)!,
+        options: [:]
+      )
+      XCTAssertTrue(result)
+
+      self.presentViewController.assertValues([4])
```

Comment:
might be nice to know what is in the stack too. (ie `CommentRepliesViewController`) here and above.

Reply:
Ah so the view model output is being mapped to the observer as a count of view controllers so I think this is the designed testing behavior.